events-api/protobuf/v1/event.proto

```diff
@@ -101,3 +106,9 @@ enum RebuildStatus {
   // Rebuild failed
   Failed = 4;
 }
+
+// Replica event details
+message ReplicaEventDetails {
+  // Pool name
+  string pool = 1;
```

Comment:
What about the replica uuid?

Reply:
Btw let's add optional pool_uuid as well

Reply:
Replica uuid is the "lvol.name()" which is added to target. I can add "lvol.uuid()" to the event details.

Reply:
Should the pool_uuid be optional?

Reply:
Ah this is coming from dataplane now, so should not be of "optional" kind.
Replica uuid being name is just current way control-plane creates replicas, this may change in the future.

Reply:
Done